SampleApp/TangramVC.swift

```diff
@@ -8,16 +8,30 @@
 
 import UIKit
 import TangramMap
-class TangramVC:  MapViewController{
+class TangramVC:  MapViewController, MapMarkerSelectDelegate {
 
   override func viewDidLoad() {
     super.viewDidLoad()
-    let _ = try? loadScene("scene.yaml")
+    markerSelectDelegate = self
+    loadSceneFileAsync("scene.yaml") { (scene) in
+      let _ = self.showCurrentLocation(true)
+      self.showFindMeButon(true)
```

Comment:
Thanks for the detailed info around this! I've added `[unowned self]` to the capture list in `TangramVC` and am now setting `onSceneLoaded` to `nil` after it is called. I agree that we should do the right thing as framework developers.